packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -337,6 +337,20 @@ function AppLayoutSidebarNavItemInner({
 					},
 
 					...focusStyles,
+					':focus-visible': {
+						// Elevate focused item above next sibling
+						zIndex: tokens.zIndex.elevated,
+						// Bring the focus ring in against the element so that it remains on screen and sibling element borders don't show in the offset
+						outlineOffset: `-${packs.outline.outlineOffset}px`,
+
+						// Make the left border appear to sit below the constrained focus ring
+						'&::before': {
```

Comment:
I don't think we should bother trying to do anything with this, because it noticeably gets a line beside it. Changing it to  the original `tokens.borderWidth.md` didn't work either
<img width="164" alt="image" src="https://github.com/user-attachments/assets/bd9c4894-2323-4ac1-9001-da07c2420a26">

Reply:
Hmm but it doesn't have a gap for top/bottom, so maybe my calculations are just off.

Reply:
I've just applied `zIndex: -1`, which is working better.